llvm/lib/Target/AMDGPU/SIPreEmitPeephole.cpp

```diff
@@ -456,22 +453,8 @@ bool SIPreEmitPeephole::removeExeczBranch(MachineInstr &MI,
 
 // If support is extended to new operations, add tests in
 // llvm/test/CodeGen/AMDGPU/unpack-non-coissue-insts-post-ra-scheduler.mir.
-bool SIPreEmitPeephole::isUnpackingSupportedInstr(MachineInstr &MI) const {
```

Comment:
Can you handle the removal of this in a separate NFC PR? I need it for something related, so I don't think it should be blocked by adding V_PK_MOV.